Core/clim-core/gadgets.lisp

```diff
@@ -520,7 +520,9 @@
                      :reader slider-number-of-quanta))
   (:documentation "The value is a real number, and default value for orientation is :vertical,
 and must never be nil.")
-  (:default-initargs :orientation :vertical))
+  (:default-initargs
+    :text-style (merge-text-styles (make-text-style :sans-serif nil nil) *default-text-style*)
```

Comment:
Please add a function like `default-gadget-text-style' so this initform doesn't have to repeated as many times.

Reply:
1. `merge-text-styles` is unnecessary here. `make-text-style` will merge with `*default-text-style*` all NIL arguments.
2. shouldn't `(:default-initargs :text-style (make-text-style :sans-serif nil nil))` be supplied as a `basic-gadget` class option? I don't see a need for `default-gadget-text-style`, but maybe I miss something (?)

Reply:
Maybe this is the wrong solution. See my other reply about the underlying issue.